libs/design-system/error-pages/not-found-page/src/lib/not-found-page.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="page-title">This page isn't available</div>
```

Comment:
No need to prefix classes with `page-`. Styles will be scoped to this component anyways so simply using classes such as `title` and `description` works fine